app/controllers/subscriptions_controller.rb

```diff
@@ -35,6 +23,18 @@ def create
     redirect_to new_subscription_path(plan: (@plan.try(:third_party_identifier) || 'premium'))
   end
 
+  def update
+    customer = Stripe::Customer.retrieve(current_user.stripe_customer_id)
+    subscription = customer.subscriptions.retrieve(customer.subscriptions.first.id)
+    subscription.plan = "premiummob"
```

Comment:
Types and symbols are less error prone that hardcoded strings ;)

Please. Consider refactoring those `premium` and `premiummob` plan types too.